app/views/layouts/_nav.html.erb

```diff
@@ -12,6 +12,7 @@
       <% nav_data.each_with_index do |children, i| %>
         <%=
           render 'layouts/nav_children',
+          pre_expand_children: false,
```

Comment:
nit, optional: "pre-expand" sounds to me like something that happens before expansion. Maybe a better name for this is `start_expanded`? I don't feel strongly about this though.

Reply:
Cool, `started_expanded` reads nicer. Just pushed up the change in 472b3b75a62219308371b30c6666a8bb7e96acbc